Tests/LootLockerTestUtils/LootLockerTestConfigurationGame.cs

```diff
@@ -0,0 +1,311 @@
+            var formattedEndpoint = string.Format(endPoint.endPoint, ActiveGameId);
+
+            string json = LootLockerJson.SerializeObject(request);
+
+            LootLockerAdminRequest.Send(formattedEndpoint, endPoint.httpMethod, json,
+                onComplete: (serverResponse) =>
+                {
+                    var leaderboardResponse = LootLockerResponse.Deserialize<LootLockerTestLeaderboard>(serverResponse);
+                    Debug.Log("KeyFind " + leaderboardResponse.key);
+                    Leaderboards.Add(leaderboardResponse);
+                    onComplete?.Invoke(leaderboardResponse);
+                }, true);
+        }
+
+        public void UpdateLeaderboard(LootLockerTestLeaderboard leaderboard, UpdateLootLockerLeaderboardRequest request, Action<bool /*success*/, string /*errorMessage*/, LootLockerTestLeaderboard /*leaderboard*/> onComplete)
+        {
+            EndPointClass endPoint = LootLockerTestConfigurationEndpoints.updateLeaderboard;
+            var formattedEndpoint = string.Format(endPoint.endPoint, ActiveGameId, leaderboard.key);
+
+            string json = LootLockerJson.SerializeObject(request);
+
+            LootLockerAdminRequest.Send(formattedEndpoint, endPoint.httpMethod, json,
+                onComplete: (serverResponse) =>
+                {
+                    var leaderboardResponse = LootLockerResponse.Deserialize<LootLockerTestLeaderboard>(serverResponse);
+                    onComplete?.Invoke(leaderboardResponse.success, leaderboardResponse?.errorData?.message, leaderboardResponse);
+                }, true);
+        }
+
+        public void UpdateLeaderboardSchedule(string leaderboardKey, UpdateLootLockerLeaderboardScheduleRequest request, Action<bool /*success*/, string /*errorMessage*/, LootLockerTestLeaderboard /*leaderboard*/> onComplete)
+        {
+            if (string.IsNullOrEmpty(LootLockerConfig.current.adminToken))
+            {
+                // Already signed in
+                onComplete?.Invoke(false, "Not logged in", null);
+                return;
+            }
+            EndPointClass endPoint = LootLockerTestConfigurationEndpoints.updateLeaderboardSchedule;
+            var formattedEndpoint = string.Format(endPoint.endPoint, ActiveGameId, leaderboardKey);
+
+            string json = LootLockerJson.SerializeObject(request);
+
+            LootLockerAdminRequest.Send(formattedEndpoint, endPoint.httpMethod, json,
+                onComplete: (serverResponse) =>
+                {
+                    var leaderboardResponse = LootLockerResponse.Deserialize<LootLockerTestLeaderboard>(serverResponse);
+                    onComplete?.Invoke(leaderboardResponse.success, leaderboardResponse?.errorData?.message, leaderboardResponse);
+                }, true);
+        }
+
+        public void AddLeaderboardReward(LootLockerTestLeaderboard leaderboard, Action<LootLockerLeaderboardDetailResponse /*leaderboard*/> onComplete)
```

Comment:
All of these methods could maybe be on the leaderboard class? I guess the issue is giving them the active game id.

Reply:
Fixed